### DISCUSSION TITLE
​feature Implementation: Preview Link Document in Popover or New Window

### DISCUSSION THREAD

**Discussion by ousc** [toeverything/AFFiNE#6190] [Ideas]
### Description

I propose the addition of a feature that allows users to preview linked documents either in a popover or in a new window. This feature aims to enhance user experience by providing immediate access to the content of linked documents without the need to navigate away from the current page, thereby preserving the user's workflow and context. It should cater to varying user preferences by offering the flexibility to preview content in a less intrusive popover or a more detailed new window view. This feature is anticipated to improve accessibility and convenience across the platform, ideally leveraging modern web technologies to ensure cross-browser and cross-device compatibility.

### Comparisons and Inspirations:
Craft offers a similar feature where documents can be previewe in a popover.

### Use case

Quick Glance through Popover: Users can quickly hover over or click on a document link to see a preview in a small overlay. This is useful for gaining immediate context or checking content without leaving the current page.
﻿
In-depth Review in a New Window: For users requiring a more detailed look or wishing to interact with the document directly (e.g., for printing or downloading), opening the document in a new window provides an uninterrupted workflow in the original context.
﻿
This functionality would be especially beneficial in environments where accessing multiple documents efficiently is crucial, in any application involving significant document reference and cross-linking.

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

[a comment by an automated account is not shown]

**Comment by ousc** [comment]
Also, I'm sorry, but I'm not sure if this feature request should be made in AFINE or Blocksuite.

**Comment by EYHN** [comment]
@ousc Thanks for the suggestion, I'll discuss it with our product team tomorrow.
